pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -46,6 +47,47 @@ static __always_inline void update_conn_state(conn_tuple_t *t, conn_stats_ts_t *
     }
 }
 
+static __always_inline bool _isTLS(conn_tuple_t *t) {
+    if (bpf_map_lookup_elem(&tls_connection, t) != NULL) {
+        return true;
+    }
+    return false;
+}
```

Comment:
```suggestion
static __always_inline bool is_tls_helper(conn_tuple_t *t) {
    return bpf_map_lookup_elem(&tls_connection, t) != NULL;
}
```

Reply:
1. the kernel has a naming convention of `lower_case_with_underscore`
2. nit change

Reply:
done

Reply:
done